### ISSUE TITLE
[Bug]: Show browser is not keeping the browser open after running the test

### ISSUE THREAD

**Issue by jespinili** [microsoft/playwright#31100]
### Version

1.44.0

### Steps to reproduce

Open Playwright Test for VSCode extension
Select/Tick the Show browser checkbox
Run any test

### Expected behavior

The browser should be kept open after the test run is completed

### Actual behavior

The browser is closed after the test is run completed

### Additional context

Playwright Test for VSCode extension version: v1.1.6

### Environment

```shell
System:
    OS: Windows 10 10.0.19045
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 17.10 GB / 31.74 GB
  Binaries:
    Node: 22.2.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.7.0 - C:\Program Files\nodejs\npm.CMD
  IDEs:
    VSCode: 1.89.1 - C:\Users\jeffreye\AppData\Local\Programs\Microsoft VS Code\bin\code.CMD
  Languages:
    Bash: 5.2.26 - C:\Program Files\Git\usr\bin\bash.EXE
  npmPackages:
    @playwright/test: ^1.44.0 => 1.44.0
```

**Comment by mxschmitt** [comment]
This might be fixed in 1.44.1 - could you try that version?

**Comment by jespinili** [comment]
Hi Max, yes it did. Thanks!
